### PR TITLE
Add Plug to Portuguese(Brazil)

### PR DESCRIPTION
**Pull request by pragmaticivan**

### CODE REVIEW

**Comment by pragmaticivan** [comment]
cc @psantos10  @brunoasantos  If you guys have time, please review :D

**Comment by pragmaticivan** [comment]
Fixed @psantos10 can I merge?

**Comment by pragmaticivan** [comment]
ping @psantos10

File: pt/lessons/specifics/plug.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
+---
+layout: page
+title: Plug
+category: specifics
+order: 1
+lang: pt
+---
+
+Se você estiver familiarizado com Ruby, você pode pensar sobre Plug como o Rack com um esguicho de Sinatra, ele fornece uma especificação para componentes de aplicação web e adaptadores para servidores web. Mesmo não fazendo parte do núcleo de Elixir, Plug é um projeto oficial de Elixir.
```
Comment (philss) [inline pt/lessons/specifics/plug.md]: I think you could use `uma pitada` instead of `um esguicho`.

**Comment by pragmaticivan** [comment]
Fixed @philss

File: pt/lessons/specifics/plug.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+    conn
+    |> put_resp_content_type("text/plain")
+    |> send_resp(200, "Hello World!")
+  end
+end
+```
+
+## Criando um Plug
+
+Para este exemplo, iremos criar um Plug para verificar se a requisição tem um conjunto de parâmetros necessários. Ao implementar a nossa validação em um Plug, podemos ter a certeza de que apenas os pedidos válidos irão ser gerenciados através de nosso aplicativo. Vamos esperar que o nosso Plug seja inicializado com duas opções: `:paths` e `:fields`. Estes irão representar os caminhos que aplicamos nossa lógica onde os campos são exigidos.
```
Comment (philss) [inline pt/lessons/specifics/plug.md]: `serão` could replace `irão ser`

File: pt/lessons/specifics/plug.md [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+end
+```
+É isso aí! Nós configuramos o nosso Plug para verificar se todas as requisições para `/upload` incluem tanto `"content"` quanto `"mimetype"`, só então o código de rota irá ser executado.
+
+Por agora nosso endpoint `/upload` não é muito útil, mas vimos como criar e integrar o nosso Plug.
+
+
+## Executando nosso Web App
+
+Antes de podermos executar nossa aplicação nós precisamos instalar e configurar o nosso servidor web, neste caso Cowboy. Por agora vamos fazer as mudanças necessárias no código para executar tudo, nós chegar a coisas específicas posteriormente.
```
Comment (philss) [inline pt/lessons/specifics/plug.md]: this section:

```
# original: For now we’ll just make the code changes necessary to run everything, we’ll dig into specifics in later lessons.

Por agora vamos fazer as mudanças necessárias no código para executar tudo, nós chegar a coisas específicas posteriormente.
```

could be something like:
```
Por agora vamos fazer as mudanças necessários no código para rodar tudo, e então vamos nos aprofundar em coisas específicas em lições futuras.
```

**Comment by pragmaticivan** [comment]
Fixed @philss can I merge?

**Comment by philss** [comment]
:shipit:

**Comment by pragmaticivan** [comment]
Thanks @philss o/

**Comment by philss** [comment]
@pragmaticivan you're welcome! o/

**Pull request merged**